recipes/role-based-access/query.json

```diff
@@ -0,0 +1,13 @@
+{
+    "dimensions": [
+        "Orders.status"
+    ],
+    "timeDimensions": [],
```

Comment:
Do we really need empty query parts here? I mean, `timeDimensions` and `filters`?

Reply:
We don't need it, you are right.